doc/modules/grid_search.rst

```diff
@@ -167,6 +171,349 @@ variable that is log-uniformly distributed between ``1e0`` and ``1e3``::
       Random search for hyper-parameter optimization,
       The Journal of Machine Learning Research (2012)
 
+.. _successive_halving_user_guide:
+
+Searching optimal parameters with successive halving
+====================================================
+
+Scikit-learn also provides the :class:`HalvingGridSearchCV` and
+:class:`HalvingRandomSearchCV` estimators that can be used to
+search a parameter space using successive halving [1]_ [2]_. Successive
+halving is an iterative selection process where all candidates (the
```

Comment:
It reminds me of a tournament... but maybe we can't really call this `GridTournamentCV`!

Reply:
This is definitely a tournament, and SH is basically a genetic algorithm without the mutation part. I think we should stick with the name from the literature, but I'll add a comment about it being a tournament